.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+| JIRA link | Slack thread (if any) | Product spec | Tech spec/One-pager | Bundle Size Difference |
```

Comment:
nit: My earlier comment might've been not clear 😅 ... Can you keep these titles in the first column and the values in the second ?

Reply:
ohh acchha, that way 😅